.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: java
 dist: trusty
 jdk:
-- openjdk7
+- openjdk8
 - oraclejdk8
```

Comment:
```suggestion
```
I don't think we need both flavors of java 8.  Let's just pick "openjdk8" since that is consistent with the java template repo and other SDK projects, etc.